website/content/docs/concepts/aliases.mdx

```diff
@@ -0,0 +1,269 @@
+---
+layout: docs
+page_title: Aliases
+description: |-
+  Aliases let you associate a string with a Boundary resource. Learn how to use an alias instead of a target ID when you connect.
+---
+
+# Aliases
+
+An alias is a resource that lets you associate a string with a specific target.
+The string must be a globally unique value which follows DNS naming constraints.
```

Comment:
Not necessarily true, as we support hostnames(e.g an alias of `webserver` as well as DNS-like strings `webserver.boundary`. May be worth rephrasing to something like: An alias is a globally unique, DNS-like string associated with a destination resource. Examples of valid aliases may be `webserver` or `webserver.boundary`.